modules/user_accounts/php/user_accounts.class.inc

```diff
@@ -40,102 +39,56 @@ class User_Accounts extends \NDB_Menu_Filter
-            'Email',
-            'Active',
-            'Pending_approval',
-        );
-        $this->query        = $query;
-        $this->order_by     = 'Username';
-        $this->group_by     = 'users.ID';
-        $this->validFilters = array(
-            'user_psc_rel.CenterID',
-            'users.UserID',
-            'users.Real_name',
-            'users.Email',
-            'users.Active',
-            'users.Examiner',
-            'users.Pending_approval',
-        );
-
-        $this->formToFilter = array(
-            'centerID'  => 'user_psc_rel.CenterID',
-            'active'    => 'users.Active',
-            'userID'    => 'users.UserID',
-            'real_name' => 'users.Real_name',
-            'email'     => 'users.Email',
-            'pending'   => 'users.Pending_approval',
-        );
+        return true;
     }
 
     /**
-     * Does the setup required for this page. By default, sets up elements
-     * that are common to every type of page. May be overridden by a specific
-     * page or specific page type.
+     * Tells the base class that this page's provisioner can support the
+     * UserProjectMatch filter.
      *
-     * @return void
+     * @return bool true
      */
-    function setup()
+    public function useProjectFilter() : bool
     {
-        parent::setup();
-
-        // create user object
-        $user =& \User::singleton();
+        return false;
+    }
 
-        // PSC
-        if ($user->hasPermission('user_accounts_multisite')) {
-            // get the list of study sites - to be replaced by the Site object
-            $list_of_sites = array();
-            $site_arr      = \Utility::getSiteList(false);
-            foreach ($site_arr as $key=>$val) {
-                $list_of_sites[$val] = $val;
-            }
-        } else {
-            // allow only to view own site data
-            $list_of_sites = array();
-            $site          = array();
-            $site_arr      = $user->getData('CenterIDs');
-            foreach ($site_arr as $key=>$val) {
-                $site[$key] = \Site::singleton($val);
-                $list_of_sites[$site[$key]->getCenterName()]
-                    = $site[$key]->getCenterName();
-            }
-        }
+    /**
+     * {@inheritDoc}
+     *
+     * @return array
+     */
+    protected function getFieldOptions() : array
+    {
         $yesNoOptions = array(
             'Y' => 'Yes',
             'N' => 'No',
         );
 
-        $this->fieldOptions = array(
-            'sites'            => $list_of_sites,
+        return array(
+            'sites'            => \Utility::getSiteList(false),
             'actives'          => $yesNoOptions,
             'pendingApprovals' => $yesNoOptions,
         );
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @return \Loris\Data\Provisioner
+     */
+    public function getBaseDataProvisioner() : \LORIS\Data\Provisioner
+    {
+        return new UserAccountsRowProvisioner();
```

Comment:
@HenriRabalais typo "UserAccountsRowProvisioner" here. should be "UserAccountRowProvisioner"